timescaledb/how-to-guides/continuous-aggregates/about-continuous-aggregates.md

```diff
@@ -202,7 +202,7 @@ For more information about setting up multi-node, see the
 
```

Comment:
@charislam This is a drive-by, but in this file I'm getting a warning about a duplicate heading on L133 & L167, it's "Materialization engine". Is that intended?

Reply:
Reading it, it seems as if we can get rid of the one on L167 and just combine it with the section above, it discusses how the invalidation threshold is handled by the materialization engine